client/src/pages/Registraction.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Form from "react-bootstrap/Form";
+
+const Registraction = () => {
+  return (
+    <> <h1 style={{ textAlign:"center",marginTop:"50px" }}>Registraction Form</h1>
+    <Form style={{ width:"500px",margin:"100px auto" }}>
```

Comment:
This looks good, can we maybe not used fixed widths, you can use the `Column` component and use some of the available breakpoints e.g `md`, `sm`